--- cpuinfo-source/CMakeLists.txt	2025-02-21 18:44:29.732684191 +0000
+++ /usr/local/google/home/dahlb/projects/backup/cpuinfo-source/CMakeLists.txt	2025-02-21 18:42:48.092778141 +0000
@@ -108,7 +108,7 @@
       "Target operating system is not specified. "
       "cpuinfo will compile, but cpuinfo_initialize() will always fail.")
   SET(CPUINFO_SUPPORTED_PLATFORM FALSE)
-ELSEIF(NOT CMAKE_SYSTEM_NAME MATCHES "^(Windows|WindowsStore|CYGWIN|MSYS|Darwin|Linux|Android|FreeBSD)$")
+ELSEIF(NOT CMAKE_SYSTEM_NAME MATCHES "^(Windows|WindowsStore|CYGWIN|MSYS|Darwin|Linux|Android|FreeBSD|Emscripten)$")
   IF(${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.14" AND NOT IS_APPLE_OS)
     MESSAGE(WARNING
       "Target operating system \"${CMAKE_SYSTEM_NAME}\" is not supported in cpuinfo. "
@@ -226,10 +226,10 @@
     ENDIF()
   ENDIF()
 
-  IF(CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
+  # IF(CMAKE_SYSTEM_NAME STREQUAL "Emscripten")
     LIST(APPEND CPUINFO_SRCS
       src/emscripten/init.c)
-  ENDIF()
+  # ENDIF()
 
   IF(CMAKE_SYSTEM_NAME STREQUAL "Linux" OR CMAKE_SYSTEM_NAME STREQUAL "Android")
     LIST(APPEND CPUINFO_SRCS
